readme.md

```diff
@@ -116,7 +135,7 @@ How is rebase different than a merge?
 
 Your Answer:
 ```text
-
+Merging combines a branch with the master (assuming you're on master and git merge branch_name) even if they go off and haven't remerged with an updated version of master. Rebasing keeps everything linear and works the branches into the master like they've always been there.
```

Comment:
:+1:
I recommend you include something about rebase rewriting the commits (or history) so that it looks "like they've always been there".